crypto/multisig/wire.go

```diff
@@ -0,0 +1,26 @@
+package multisig
+
+import (
+	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+)
+
+// TODO: Figure out API for others to either add their own pubkey types, or
+// to make verify / marshal accept a cdc.
```

Comment:
Huh. We don't really expose this as an API at all, which perhaps suggests that PubKey really could be a `oneof`

Reply:
Yes I wasn't suggesting you do anything. But it goes back to the larger conversation of "do we really need pubkey to be an any-one-can-implement-it-without-permission interface" ie. the kind of thing Amino was built for, or would pure protobuf (using protobufs union type, called `oneof`) do just fine, since there isn't really a nice way to add new pubkey types right now without changing our crypto lib

Note this is unlike Msg in the SDK which really does want to be an interface.

Reply:
Oh, I think pure protobuf is better here. We have to register it in all the codecs, so its not really like anyone can just implement it.